README.md

```diff
@@ -364,7 +364,8 @@ var jobs = {
 
 **notes**
 
-- All plugins which return `(error, toRun)`.  if `toRun = false` on  `beforeEnqueue`, the job begin enqueued will be thrown away, and if `toRun = false` on `beforePerfporm`, the job will be reEnqued and not run at this time.  However, it doesn't really matter what `toRun` returns on the `after` hooks.
+- All plugins run a callback with arguments `(error, toRun)`.  if `toRun = false` on  `before_enqueue`, the job being enqueued will be thrown away, and if `toRun = false` on `before_perform`, the job will be reEnqueued and not run at this time. `toRun = false` does not affect execution in the `after_enqueue` hook.
+- When running multiple plugins on a job (with the array syntax), the plugins will be run sequentially (from let to right). If `toRun = false` in the `after_perform` hook, the next plugin will not be executed. Otherwise, `toRun` has no effect in the execution flow.
```

Comment:
Are you sure?

I think we should make the distinction between  "hooks"  and "plugins". It prevents the next *same-level* hook from executing.

To be more accurate (and for rest of mind), I made [an excel chart](https://docs.google.com/a/borna-almasi.com/spreadsheets/d/1vf6yEy5oMzPMd4PI-q572rGAOkei2-mI3rsjCMREd88/edit?usp=sharing) that outlines exactly what happens when you pass toRun = false or callback with an error

I encourage anyone making plugins to check it out.

One thing I noticed is that the queue is empty even though I returned `toReturn = false`